### ISSUE TITLE
Implement method of moments

### ISSUE THREAD

**Issue by juliohm** [JuliaEarth/ExtremeStats.jl#2]
Maximum likelihood fits aren't very robust with GEV distributions. It would be nice to implement the method of moments as an alternative.

**Comment by davidanthoff** [comment]
Are there similar problems with the GP distribution, or is that part more robust?

**Comment by juliohm** [comment]
Hi @davidanthoff,

I'd have to investigate the theory more carefully to answer it properly. Are you having issues with the GP estimation as well?

**Comment by davidanthoff** [comment]
No, I'm just starting to play with this, and was wondering whether there was a known problem or not :)

Also, there is https://github.com/jojal5/Extremes.jl, which seems to have MM and some bayesian estimator already? Are there any efforts to merge the two packages? They really seem to do the same thing, right?

**Comment by juliohm** [comment]
Thank you for sharing the package. I know the author, will ask if he is interested in joining efforts here in the org.

**Comment by davidanthoff** [comment]
I did try the fit for the GP in both packages, got exactly the same results, so I guess that is a good sign :)

**Comment by juliohm** [comment]
That is promising at least :)

I hope to get back to extreme value theory at some point. Currently, my research is too focused on GeoStats.jl, and that is demanding a great amount of :hourglass:

**Comment by Datseris** [comment]
In this library, I have implemented method of moments and probability weighted moments as ways to fit a GPD to data produced using the Peak over Threshold approach: 

https://github.com/JuliaDynamics/FractalDimensions.jl/blob/main/src/extremes_based/gpd.jl#L33-L47

The implementations come from the paper: Chaos 33, 073143 (2023) https://doi.org/10.1063/5.0152370

You are interested in section V.A. of the paper, starting at page 6. It shows efficacy of estimators from data sampled directly from a GPD. It shows that maximum likelyhood methods are surprisingly not very good.

**Comment by Datseris** [comment]
(p.s. i'll probably add a method using your package in our FractalDimensions.jl package)

**Comment by juliohm** [comment]
It would be awesome to add the method of moments here as well @Datseris so that it reaches a broader audience ❤️
